src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -424,6 +424,11 @@ void runtime::cpu::CPU_ExternalFunction::compile()
         writer << "#include <tbb/flow_graph.h>";
     }
 
+#ifdef NGRAPH_DISTRIBUTED
+    writer << "#include <mlsl.hpp>\n";
+    writer << "#define NGRAPH_DISTRIBUTED\n";
```

Comment:
is this needed?

Reply:
Yes it is needed since compiled function uses CPURuntimeContext struct which has MLSL related members placed under `#ifdef NGRAPH_DISTRIBUTED`